lib/src/components/NxList/__tests__/NxList.test.tsx

```diff
@@ -4,192 +4,216 @@
   });
 
-  it('shows the emptyMessage when there are no children, no error, and not loading', async function() {
+  it('renders children correctly', function() {
+    const children = <NxList.Item>Cat</NxList.Item>,
+        view = quickRender({ children }),
+        list = view.getByRole('list'),
+        listItem = view.getByRole('listitem');
 
-    const trulyEmptyComponent = await mountAttached(<NxList emptyMessage="Empty message"></NxList>),
-        emptyListComponent = await mountAttached(<NxList emptyMessage="Empty message">{[]}</NxList>);
+    expect(list).toBeInTheDocument();
+    expect(listItem).toBeInTheDocument();
+    expect(list).toContainElement(listItem);
+    expect(listItem).toHaveTextContent('Cat');
+  });
 
-    // the enzyme wrapper contains the <NxList> as its top element, the native el is one level down
-    expect(trulyEmptyComponent.children()).toMatchSelector('ul');
-    expect(trulyEmptyComponent).toContainExactlyOneMatchingElement('li.nx-list__item');
-    expect(trulyEmptyComponent.find('span')).toHaveText('Empty message');
+  describe('emptyMessage', function() {
+    describe('when not provided', function() {
+      it('shows the default empty message when there are no children, ' +
+        'no error, and not loading', function() {
+        // truly empty component
+        const { getByRole, rerender } = quickRender();
 
-    expect(emptyListComponent.children()).toMatchSelector('ul');
-    expect(emptyListComponent).toContainExactlyOneMatchingElement('li.nx-list__item');
-    expect(emptyListComponent.find('span')).toHaveText('Empty message');
-  });
+        expect(getByRole('list')).toHaveTextContent('This list is empty.');
+        expect(getByRole('listitem')).toHaveTextContent('This list is empty.');
 
-  it('does not show the emptyMessage when there are children', async function() {
-    const component = await mountAttached(
-      <NxList emptyMessage="Empty message">
-        <NxList.Item>
-          <NxList.Text>Foo</NxList.Text>
-        </NxList.Item>
-      </NxList>
-    );
+        // empty list component
+        rerender(<NxList>{[]}</NxList>);
 
-    expect(component).not.toIncludeText('Empty message');
-  });
+        expect(getByRole('list')).toHaveTextContent('This list is empty.');
+        expect(getByRole('listitem')).toHaveTextContent('This list is empty.');
+      });
 
-  it('does not show the emptyMessage when isLoading', async function() {
-    const component = await mountAttached(<NxList emptyMessage="Empty message" isLoading />);
+      it('does not show the default empty message when there are children', function() {
+        const children = <NxList.Item>Cat</NxList.Item>,
+            { getByRole } = quickRender({ children });
 
-    expect(component).not.toIncludeText('Empty message');
-  });
+        expect(getByRole('list')).not.toHaveTextContent('This list is empty.');
+        expect(getByRole('listitem')).not.toHaveTextContent('This list is empty.');
```

Comment:
I think checking the list item is redundant since you just checked the entire list of the line above
```suggestion
```